### PR TITLE
[sftp] create directories

### LINKED ISSUE #4036: [lib-sftp] Create alma directory structure with ansible
- [ ] /alma/recap
- [ ] /alma/publishing
- [ ] /alma/bursar
- [ ] /alma/fund_adjustment
- [ ] /alma/scsb_renewals
- [ ] /alma/people
- [ ] /alma/pod
- [ ] /alma/invoice_status
- [ ] /alma/invoices
- [ ] /alma/aspace (it's already there but check again)
- [ ] /alma/datasync_processing

### PR DESCRIPTION
**Pull request by kayiwa**
ensure presence of alphabetized directories

will close #4036

### CODE REVIEW

File: roles/system_ldap/tasks/main.yml [hunk 1]
```diff
@@ -46,12 +46,23 @@
...
     recurse: true
     owner: "{{ almasftp_user }}"
     group: "domain users@pu.win.princeton.edu"
     mode: 0775
+  loop:
+    - "/alma/bursar"
+    - "/alma/datasync_processing"
+    - "/alma/fund_adjustment"
+    - "/alma/invoice_status"
+    - " /alma/invoices"
```
Comment (acozine) [inline roles/system_ldap/tasks/main.yml:60]:
```suggestion
    - "/alma/invoices"
```

**Comment by acozine** [comment]
Current failure in CI is:
```
TASK [ezproxy : Ezproxy | add geoip ppa] ***************************************
  fatal: [instance]: FAILED! => {"changed": false, "cmd": "apt-key adv --recv-keys --no-tty --keyserver hkp://keyserver.ubuntu.com:80 F44B38CE3DB1BF64B61DBD28DE1997DCDE742AFA", "msg": "Warning: apt-key output should not be parsed (stdout is not a terminal)\ngpg: keyserver receive failed: Cannot assign requested address", "rc": 2, "stderr": "Warning: apt-key output should not be parsed (stdout is not a terminal)\ngpg: keyserver receive failed: Cannot assign requested address\n", "stderr_lines": ["Warning: apt-key output should not be parsed (stdout is not a terminal)", "gpg: keyserver receive failed: Cannot assign requested address"], "stdout": "Executing: /tmp/apt-key-gpghome.LlObf3oDQK/gpg.1.sh --recv-keys --no-tty --keyserver hkp://keyserver.ubuntu.com:80 F44B38CE3DB1BF64B61DBD28DE1997DCDE742AFA\n", "stdout_lines": ["Executing: /tmp/apt-key-gpghome.LlObf3oDQK/gpg.1.sh --recv-keys --no-tty --keyserver hkp://keyserver.ubuntu.com:80 F44B38CE3DB1BF64B61DBD28DE1997DCDE742AFA"]}
```

**Review by christinach — APPROVED** [review]

**Comment by acozine** [comment]
Cherry-pick from #4014.

**Comment by acozine** [comment]
I started the cherry-picking process by updating the bibdata tests, which were failing most recently. We'll see what fails next . . .

**Pull request merged**